.gitignore

```diff
@@ -14,8 +14,6 @@ draft-patton-cfrg-vdaf.xml
 lib
 report.xml
 venv/
-sagelib
-*.sage.py
```

Comment:
Man, remember when we we're just had a bunch of `.sage` files? That was truly terrible. Thank you so much for your hard work on the software engineering for this spec.